### PR TITLE
switched to official elasticsearch container images v6.5.4

### PR DESCRIPTION
**Pull request by maxwelllincoln**

### CODE REVIEW

**Comment by emailbob** [comment]
test this please

**Comment by emailbob** [comment]
test this please

**Comment by maxwelllincoln** [comment]
test this please

**Comment by maxwelllincoln** [comment]
build this please

**Comment by maxwelllincoln** [comment]
test this please

**Comment by maxwelllincoln** [comment]
test this please

**Comment by maxwelllincoln** [comment]
test this please

**Comment by maxwelllincoln** [comment]
test this please

**Comment by emailbob** [comment]
test this please

**Comment by emailbob** [comment]
test this please

**Comment by maxwelllincoln** [comment]
test this please

**Comment by maxwelllincoln** [comment]
test this please

**Comment by emailbob** [comment]
test this please

**Comment by maxwelllincoln** [comment]
test this please

**Comment by maxwelllincoln** [comment]
test this please

**Comment by maxwelllincoln** [comment]
test this please

**Comment by emailbob** [comment]
test this please

**Comment by chrislovecnm** [comment]
Test this please

**Comment by maxwelllincoln** [comment]
test this please

**Comment by maxwelllincoln** [comment]
test this please

**Comment by maxwelllincoln** [comment]
test this please

File: elasticsearch/manifests/configmap.yaml [hunk 1]
```diff
@@ -31,7 +31,8 @@ data:
   post-start-hook.sh: |-
     #!/bin/bash
     set -e
-    apk --no-cache add jq
+    yum install -y epel-release
```
Comment (chrislovecnm) [inline elasticsearch/manifests/configmap.yaml]: Is jq installed? I really hate this code here.

File: scripts/load-shakespeare.sh [hunk 2]
```diff
@@ -41,10 +41,11 @@ curl "http://localhost:9200/shakespeare" \
...
+echo " "
 echo "Loading the data into the Elasticsearch cluster"
-curl "http://localhost:9200/shakespeare/doc/_bulk" \
-   -s \
-   -H "Content-Type: application/x-ndjson" \
-   -X POST \
+
+curl "http://localhost:9200/shakespeare/doc/_bulk?pretty" \
+    -H "Content-Type: application/x-ndjson" \
+   --retry 5 \
```
Comment (chrislovecnm) [inline scripts/load-shakespeare.sh]: Why do we need this?

File: elasticsearch/manifests/configmap.yaml (hunk 1 above)
Comment (maxwelllincoln) [reply elasticsearch/manifests/configmap.yaml]: `jq` isn't already installed in the elasticsearch container:



I took out the `jq` bits in that shut down script replaced with them with some inline python.

File: scripts/load-shakespeare.sh (hunk 2 above)
Comment (maxwelllincoln) [reply scripts/load-shakespeare.sh]:
```
curl "http://localhost:9200/shakespeare/doc/_bulk?pretty" \
    -H "Content-Type: application/x-ndjson" \
   --retry 5 \
   --data-binary @"$PROJECT_ROOT"/elasticsearch/data/shakespeare_6.0.json
```

This `curl` command makes a POST request to upload some data to populate the index mapping that is created earlier in this file. This data is used validate that the cluster is working as intended, e.g the ingest nodes are reachable and that the cluster indexes are sharded across all of the nodes.

Comment (chrislovecnm) [reply scripts/load-shakespeare.sh]: Right. Was not clear. Why do we need the retry?

**Comment by chrislovecnm** [comment]
Well CI started again.

File: scripts/load-shakespeare.sh (hunk 2 above)
Comment (chrislovecnm) [reply scripts/load-shakespeare.sh]: Let’s merge this beast of a PR and see if we can remove the retry

**Comment by maxwelllincoln** [comment]
test this please

**Comment by maxwelllincoln** [comment]
test this please

**Pull request merged**